app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -63,15 +63,35 @@ class RefereeProfile extends Component {
     } = this.state
 
     return {
-      firstName: changedFirstName || referee.firstName,
-      lastName: changedLastName || referee.lastName,
+      firstName: changedFirstName !== null ? changedFirstName : referee.firstName,
```

Comment:
Let's use the affirmative assertion here, potentially with lodash. So something like `_.isNull(changedFirstName) ? referee.firstName : changedFirstName`